packages/repository/src/relations/relation.helpers.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import * as _ from 'lodash';
+import {Entity, EntityCrudRepository, Filter, Options, Where} from '..';
+
+/**
+ * Finds model instances that contain any of the provided foreign key values.
+ *
+ * @param targetRepository - The target repository where the model instances are found
+ * @param fkName - Name of the foreign key
+ * @param fkValues - Array of the values of the foreign keys to be included
+ * @param scope - Additional scope constraints (not currently supported)
+ * @param options - Options for the operations
+ */
+export async function findByForeignKeys<
+  Target extends Entity,
+  TargetID,
```

Comment:
Do we really need this? Use `unknown` is probably good enough.

Reply:
`TargetID`? but we use it in [`targetRepository: EntityCrudRepository<Target, TargetID, TargetRelations>,`](https://github.com/strongloop/loopback-next/pull/3473/files#diff-791ac353d48a6890b8c35dfaa1c836a1R24)

Reply:
We can remove `TargetID` from the type params and use `unknown`:

 targetRepository: EntityCrudRepository<Target, unknown, TargetRelations>,

Reply:
Ohh I see, thanks! What's the benefit of using `unknown`?

Reply:
The `TargetID` does not matter in our usage of `find` as no `findById` is involved.